### PR TITLE
fix(all-dashboards-page): fix data state for no-data case

### PR DESCRIPTION
**Pull request by sulmoJ**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [x] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [x] `Error / Warning / Lint / Type`

### Description
SSIA

### CODE REVIEW

**Review by yuda110 — APPROVED** [review]
LGTM!

**Review by WANZARGEN — APPROVED** [review]
LGTM!

File: src/services/dashboards/all-dashboards/AllDashboardsPage.vue [hunk 1]
```diff
@@ -111,6 +109,15 @@ export default {
...
             dashboardTotalCount: computed(() => store.getters['dashboard/getDashboardCount']),
+            filteredDashboardStatus: computed(() => {
+                if (state.scopeStatus === SCOPE_TYPE.DOMAIN) {
+                    return !!(state.workspaceDashboardList.length);
+                }
+                if (state.scopeStatus === SCOPE_TYPE.PROJECT) {
+                    return !!(state.projectDashboardList.length);
+                }
+                return !!(state.dashboardTotalCount);
+            }),
```
Comment (piggggggggy) [inline src/services/dashboards/all-dashboards/AllDashboardsPage.vue:120]: praise: verrrrrrrry clear~!

**Review by piggggggggy — APPROVED** [review]
LGTM!!!!

**Pull request merged**